docs/en/deployment/third-party-tool-btpanel.md

```diff
@@ -0,0 +1,73 @@
+## aaPanel Docker One-Click Install
+
+Go to [aaPanel official website](https://www.aapanel.com/new/download.html), Select the script to download and install(Skip this step if you already have it installed)
+
+# Deploy Apollo using aaPanel
+
+## Prerequisite
+
+To install aaPanel, go to the [aaPanel](https://www.aapanel.com/new/download.html#install) official website and select the corresponding script to download and install.
+
+## Deployment
+
+aaPanel(Applicable versions 7.0.11 and above) Deployment guidelines
+
+1. Log in to aaPanel and click `Docker` in the menu bar
+
+   ![Docker](en/images/deployment/btpanel/install.png)
```

Comment:
Does it really work in your environment? It doesn't work for me.

![image](https://github.com/user-attachments/assets/d3e0fb79-d689-4d8e-aaf1-ff29612a69b7)

I suppose it should be fixed like this

```suggestion
   ![Docker](../images/deployment/btpanel/install.png)
```